### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by maaslalani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
-# Soft-Serve Action
+# Soft Serve Action
 
-A Github Action that synchronize repositories to your own [soft-serve](https://github.com/charmbracelet/soft-serve) server.
+![soft-serve-action-banner](https://user-images.githubusercontent.com/42545625/199863722-1a195d17-43f7-4775-b1d9-d2d178b867b2.png)
+
+Synchronize GitHub repositories to your own self-hosted [Soft Serve](https://github.com/charmbracelet/soft-serve) git server on every push with GitHub actions.
```
Comment (maaslalani) [inline README.md:5]:
```suggestion
Synchronize GitHub repositories to your own [Soft Serve](https://github.com/charmbracelet/soft-serve) server on every push with GitHub actions. By the way, [Soft Serve](https://github.com/charmbracelet/soft-serve) is a self-hostable Git server for the command line.
```

File: README.md [hunk 2]
```diff
@@ -22,24 +25,44 @@ jobs:
...
+We’d love to hear your thoughts on this project. Feel free to drop us a note!
+
+* [Twitter](https://twitter.com/charmcli)
+* [The Fediverse](https://mastodon.social/@charmcli)
+* [Discord](https://charm.sh/chat)
+
 ## License
 
-[MIT](https://github.com/charmbracelet/soft-serve-action/raw/master/LICENSE)
+[MIT](https://github.com/charmbracelet/vhs/raw/main/LICENSE)
```
Comment (aymanbagabas) [inline README.md:54]: I think you meant `soft-serve` here

Comment (maaslalani) [reply README.md:54]: Ah yes, copy pasta error!

File: README.md [hunk 3]
```diff
@@ -22,24 +25,44 @@ jobs:
...
+          # Repository name on Soft Serve (defaults to GitHub repository name)
+          name: foobar
+          # The server hosting Soft Serve
+          server: my.yummy.server.srv
+          # Required only if Soft Serve is configured with authentication
+          ssh-key: ${{ secrets.SOFT_SERVE_KEY }}
+          # SSH user name, defaults to: git
+          ssh-user: abc
+          # Port on which the SSH server is running, defaults to: 22
+          ssh-port: 23231
```
[the remark this answers is not in the document]
Comment (maaslalani) [reply README.md:40]: I added the mirror option after the fact on `main`

**Pull request merged**